custom-steps/pango/version-info.json

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
I notice some files don't have the same number of fields as the others, eg: copyright field. I assume that's intended?

Reply:
It is.  My understanding is that copyright metadata is not required in these dlls.  We do have to attribute them in various ways in the various products, depending on their licenses.  I have added copyright information where it was obvious, but excluded it elsewhere where it was not.